integration_tests/integration/visitsByDate.cy.ts

```diff
@@ -14,22 +14,32 @@ context('View visits by date', () => {
   const tomorrowShortFormat = format(add(today, { days: 1 }), shortDateFormat)
   const tomorrowLongFormat = format(add(today, { days: 1 }), longDateFormat)
 
-  const prisonerNumber1 = 'A1234BC'
-  const prisonerNumber2 = 'A1592EC'
-
-  const childDob = format(sub(today, { years: 5 }), shortDateFormat)
-  const contacts = [
-    TestData.contact(),
-    TestData.contact({
-      personId: 4322,
-      firstName: 'Bob',
-      dateOfBirth: childDob,
-      relationshipCode: 'SON',
-      relationshipDescription: 'Son',
+  const prisonId = 'HEI'
+
+  const sessionSchedule = [
```

Comment:
This integration test looks to have a lot of changes but it's mainly new mock APIs - actual assertions the same with a bit of tidying